src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -321,6 +321,7 @@ public MimeMessage sendRegistrationInviteToInstructor(String courseId, Instructo
         
     }
     
+    @SuppressWarnings("deprecation")
     public String sendJoinLinkToNewInstructor(InstructorAttributes instructor, String shortName, String institute) 
```

Comment:
Actually, even though the function to generate the join link is deprecated, isn't it used all the time?